R/derive_locf_records.R

```diff
@@ -137,11 +189,15 @@ derive_locf_records <- function(dataset,
   )
 
 
-  #### Prepping 'dataset_ref' ####
+  # Setting id_vars_ref to all the variables of dataset_ref when not specified by user #
+  if (is.null(id_vars_ref)) {
+    id_vars_ref <- lapply(names(dataset_ref), sym)
+  }
 
 
-  # Get the IDs from input dataset for which the expected observations are to be added
+  #### Prepping 'dataset_ref' ####
```

Comment:
Could you update all comments according to our [comment convention](https://pharmaverse.github.io/admiraldev/dev/articles/programming_strategy.html#comments)? I.e., only a single `#` at the start. (It seems that this code was written before we agreed on the convention.)